vulture/whitelists/config_whitelist.py

```diff
@@ -0,0 +1,11 @@
+from typing import List, Optional
+from typing.io import TextIO
+from vulture import config
+
+List
+Optional
+TextIO
+
+# NOTE: This was added to make unit-tests pass and allow commiting intermediate
+# work. It should be removed once the TOML parsing is fully implemented.
+config._parse_toml
```

Comment:
Do we still need this file? I guess you just haven't removed all traces of the `typing` code, yet.

Reply:
You're right. Addressed in 06ebedd